Feature/read costume approval board for member/costume approval board

hyeneung opened the pull request:
## #️⃣연관된 이슈
- this closes #46 

## 📝작업 내용
- 출연자 의상 컨펌 단일 조회
- API 테스트 완료

hyeneung committed:
feature: costume approval board entity 작성

hyeneung committed:
feature: 출연자 단일 조회 dto 및 entity mapper 작성

hyeneung committed:
feature: not found exception 처리

hyeneung committed:
feature: 단건 조회 repository 작성

hyeneung committed:
feature: 출연자 의상 컨펌 단건 조회 기능 정의 및 구현

hyeneung committed:
feature: controller 작성, 잘못된 폴더 구조 수정

hyeneung committed:
Merge branch 'feature/read-costume-approval-board-for-member/costume-approval-board' of https://github.com/pangseon/Extra into feature/read-costume-approval-board

pangseon reviewed (changes requested):
단건조회를 위해서는 레코드의 id 값을 통해서 조회해야 합니다~ 관련내용 참고해서 작업해주세요~

hyeneung committed:
feature: response dto에 id 추가

shin0112 reviewed (approved):
수고하셨습니다!

pangseon reviewed (commented):
굿

hyeneung committed:
style: merge conflict 해결

Merge remote-tracking branch 'origin/develop' into feature/read-costume-approval-board-for-member/costume-approval-board
# Conflicts:
#	src/main/java/com/example/extra/domain/costumeapprovalboard/CostumeApprovalBoard/CostumeApprovalBoardController.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/entity/CostumeApprovalBoard.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/exception/CostumeApprovalBoardErrorCode.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/repository/CostumeApprovalBoardRepository.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/service/CostumeApprovalBoardService.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/service/impl/CostumeApprovalBoardServiceImpl.java

pangseon reviewed (approved):
성공?
